docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1612,9 +1612,10 @@ The `ActionProfile` message includes the following fields:
 
 * `max_group_size`, an `int32` representing the maximum sum of all member
   weights within any given selector group, in the case of an Action Selector, or
-  0 for an Action Profile. PSA programs can use the `@max_group_size` annotation
-  to provide this value for Action Selectors. If the annotation is omitted, the
-  P4Info field will default to 0.
+  0 for an Action Profile. `max_group_size` is never larger than `size`.
```

Comment:
This is orthogonal to your change, but while we're at it, can we discuss Action Profiles and Action Selectors in the same order as in the previous paragraph? Changing the ordering here is quite confusing

Reply:
Sure. Can do.